### PR TITLE
added support for sysctl

### PR DESCRIPTION
**Pull request by 927589452**
this is just copy paste and move from acpi to sysctl for FreeBSD

### CODE REVIEW

**Comment by 927589452** [comment]
Does this look good for you 
@neuralsan

File: plugins/battery/battery.plugin.zsh [hunk 1]
```diff
@@ -64,6 +67,53 @@ if [[ "$OSTYPE" = darwin* ]] ; then
...
+				elif [ $b -gt 20 ] ; then
+						color='yellow'
+				else
+						color='red'
+				fi
+				echo "%{$fg[$color]%}$(battery_pct_remaining)%%%{$reset_color%}"
+		else
+				echo "∞"
+		fi
+}
```
Comment (bric3) [inline plugins/battery/battery.plugin.zsh]: Mixing tabs in a space based file ;)

File: plugins/battery/battery.plugin.zsh [hunk 2]
```diff
@@ -64,6 +67,53 @@ if [[ "$OSTYPE" = darwin* ]] ; then
...
+		remaining_time=$(sysctl -n hw.acpi.battery.time)
+		if [[ $remaining_time -ge 0 ]] ; then
+				# calculation from https://www.unix.com/shell-programming-and-scripting/23695-convert-minutes-hours-minutes-seconds.html
+				((hour=$remaining_time/60))
+				((minute=$remaining_time-$hour*60))
+				echo $hour:$minute
+		fi
+}
+
+function battery_pct_prompt() {
```
Comment (bric3) [inline plugins/battery/battery.plugin.zsh]: Functions are not correctly indented, as it with the rest of the script.

Comment (927589452) [reply plugins/battery/battery.plugin.zsh]: fixed

File: plugins/battery/battery.plugin.zsh (hunk 1 above)
Comment (927589452) [reply plugins/battery/battery.plugin.zsh]: ups ^^
fixed

**Pull request merged**